governance/pyth_staking_sdk/src/pyth-staking-client.ts

```diff
@@ -254,6 +254,15 @@ export class PythStakingClient {
       );
     }
 
+    instructions.push(
```

Comment:
doesn't really matter but might be better to do it after the create position

Reply:
sure